src/grandorgue/dialogs/settings/GOSettingsOptions.cpp

```diff
@@ -407,6 +407,16 @@ GOSettingsOptions::GOSettingsOptions(GOConfig &settings, wxWindow *parent)
   item6->Add(grid, 0, wxEXPAND | wxALL, 5);
   item9->Add(item6, 0, wxEXPAND | wxALL, 5);
 
+  item6 = new wxStaticBoxSizer(wxVERTICAL, this, _("&Updates"));
+  item9->Add(item6, 0, wxEXPAND | wxALL, 5);
+  item6->Add(
+    m_CheckForUpdatesAtStartup = new wxCheckBox(
+      this, ID_COMPRESS_CACHE, _("check for updates at startup")),
```

Comment:
ID_COMPRESS_CACHE is a bad identifier here. Either introduce a new one or use wxID_ANY if it is not necessary.